lib/fathead/mdn_css/parse.pl

```diff
@@ -296,7 +296,7 @@ sub create_redirects {
 
     # Capture content inside parentheses
     # E.g. "::before (:before)"
-    if ( $title =~ m/(::.+) \((:.+)\)/ ) {
+    if ( $title =~ m/(.+) \((:.+|-webkit.+)\)/ ) {
```

Comment:
@moollaza I have a doubt, the RHS for `-moz-placeholder R` has `:-moz-placeholder` and `::-moz-placeholder`, which are different. Are they still considered duplicates?